admin_manual/configuration/server/background_jobs_configuration.rst

```diff
@@ -123,12 +120,10 @@ CleanupChunks
 ^^^^^^^^^^^^^
 
 The CleanupChunks job, contained in ``OCA\\DAV\\Command\\CleanupChunks``, will clean up outdated chunks (uploaded files) more than a certain number of days old.
-By default, the command cleans up chunks more than two days old. However, the command also accepts the number of days as an argument.
-It can be run, as follows, using the OCC command
-
-::
-
-  occ dav:cleanup-chunks
+   
+.. note:: 
+   This command needs to be added to your crontab. 
```

Comment:
No need to mention the `CleanupChunks` class name above as it's of no use anywhere since not in the oc_jobs table.

However it would be useful to have the mention of `occ dav:cleanup-chunks` since it's the actual command to add to crontab, please bring the red block above back.